social_motion_planner:
  ros__parameters:
    # Parameters for human_track_publisher.cpp
    human_tracker:
      det_obj_topic: "/zed3/zed_node/odom"
      human_track_topic: "/human/interpolated_history"
      odom_topic: "/zed3/zed_node/odom"
      pub_frame_id: "/map"
      pub_frame_rate: 15.0
      max_history_length: 12
      delay_tolerance: 3.0
      max_interp_interval: 0.4 
    # Parameters for robot_track_publisher.cpp
    robot_tracker:
      odom_topic: "/zed3/zed_node/odom"
      robot_track_topic: /robot/ego_trajectory
      pub_frame_id: "map"
      pub_frame_rate: 15.0
      max_history_length: 12
      max_interp_interval: 0.4
    